tests/conftest.py

```diff
@@ -108,10 +101,10 @@
 etopo1_unavail = pytest.mark.skipif(not const.ETOPO1_AVAILABLE,
                    reason='Skipping tests that require access to ETOPO1 data')
 
-testdata_unavail = pytest.mark.skipif(not TESTDATA_AVAIL,
+data_unavail = pytest.mark.skipif(not TESTDATA_AVAIL,
                     reason='Skipping tests that require testdata-minimal.')
```

Comment:
Perhaps this marker could be removed completely. It is not used in tests consistently and we should assume that testdata is available whenever CI is run

Reply:
But does not necessarily need to be done in this PR :)

Reply:
maybe a future PR